### PR TITLE
sys-auth/yubikey-personalization-gui: version bump to 3.1.25

### PR DESCRIPTION
**Pull request by jok-ts**
Also, port to Qt5 and EAPI 6.

Bug: https://bugs.gentoo.org/633444

### CODE REVIEW

File: sys-auth/yubikey-personalization-gui/yubikey-personalization-gui-3.1.25.ebuild [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic qmake-utils
```
Comment (a17r) [inline sys-auth/yubikey-personalization-gui/yubikey-personalization-gui-3.1.25.ebuild]: missing desktop.eclass (you are using `doicon` and `domenu` in `src_install`

File: sys-auth/yubikey-personalization-gui/yubikey-personalization-gui-3.1.25.ebuild [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+SRC_URI="https://github.com/Yubico/yubikey-personalization-gui/archive/${P}.tar.gz"
+HOMEPAGE="https://github.com/Yubico/yubikey-personalization-gui"
+
+KEYWORDS="~amd64"
+SLOT="0"
+LICENSE="BSD-2"
+IUSE="debug test"
+
+RDEPEND="
+	>=sys-auth/ykpers-1.14.0
```
Comment (a17r) [inline sys-auth/yubikey-personalization-gui/yubikey-personalization-gui-3.1.25.ebuild]: Please keep RDEPENDs sorted, and optional deps below permanent (same for DEPEND).

File: sys-auth/yubikey-personalization-gui/yubikey-personalization-gui-3.1.25.ebuild [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+DOCS=( NEWS README )
+
+src_prepare() {
+	if ! use test ; then
+		sed -i YKPersonalization.pro \
+			-e 's/src \\/src/' \
+			-e '/tests/d' || die
+	fi
+
+	default
```
Comment (a17r) [inline sys-auth/yubikey-personalization-gui/yubikey-personalization-gui-3.1.25.ebuild]: ideally you run `default` before messing with build system files, that way potential patches, be it your own or user patches, should not be harmed by a context change.

File: sys-auth/yubikey-personalization-gui/yubikey-personalization-gui-3.1.25.ebuild [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+	>=sys-auth/libyubikey-1.6
+	dev-qt/qtcore:5
+	dev-qt/qtwidgets:5
+	dev-libs/glib:2
+	virtual/libusb:1"
+DEPEND="${RDEPEND}
+	test? ( dev-qt/qttest:5 )
+	virtual/pkgconfig"
+
+DOCS=( NEWS README )
```
Comment (a17r) [inline sys-auth/yubikey-personalization-gui/yubikey-personalization-gui-3.1.25.ebuild]: Please check if this is even needed, those two files would be installed by einstalldocs automatically.

File: sys-auth/yubikey-personalization-gui/yubikey-personalization-gui-3.1.25.ebuild (hunk 1 above)
Comment (jok-ts) [reply sys-auth/yubikey-personalization-gui/yubikey-personalization-gui-3.1.25.ebuild]: That's from your diff... ;) 
I'll see to these changes.

File: sys-auth/yubikey-personalization-gui/yubikey-personalization-gui-3.1.25.ebuild (hunk 4 above)
Comment (jok-ts) [reply sys-auth/yubikey-personalization-gui/yubikey-personalization-gui-3.1.25.ebuild]: It doesn't seem to be picking those files with or without DOCS, removing the line.

File: sys-auth/yubikey-personalization-gui/Manifest [hunk 5]
```diff
@@ -1,2 +1,3 @@
 DIST yubikey-personalization-gui-3.1.23.tar.gz 1450340 BLAKE2B fb183d939334b13faf1464dc65a0a03180a2b402ae628f556dba7211e66e89628718e797a86189f654ca88225c3b834df67b497e7a94d0e210edc433bfff49b8 SHA512 912bcbcac5cad285c43995f646de6aa68cc99a4d047b7632786cd93e38c031e2fb8dcc4f57f45647ba3e276de03aaea6ed9757f72ec2c42a43780c19edcca51b
 DIST yubikey-personalization-gui-3.1.24.tar.gz 947135 BLAKE2B 846fa6792c71b57ffb99c548efe6780571c1d52061393805f0f9a76a7e7722b3fc2851ead913b09028d7798ae7bab4670260cae63a98ef0c0d1b176266fa4338 SHA512 149237a719277a6571916473d31e6099c3f3a2003fcb2893c2becee80788102496dbbc8e424f75fe1f1dd3ac87122bd35b6908cf784458635069d89e7e706cfa
+DIST yubikey-personalization-gui-3.1.25.tar.gz 951145 BLAKE2B f500ef4f591ed48d660503fe9f85e71ff19efc6b6dfef7906684e0167ca5f4f77f007541911ac62efb568022777fd620d0b368fc6f1894fd1883c22f180716ab SHA512 1171bca2caecf998dda57b76a9e11254ee976568f303fa500b01c3d73ec4e0bf39ef71928f1f87daed450f7be137cb90be0cd08aeb9a9ceb0450798de91ae8f2
```
Comment (a17r) [inline sys-auth/yubikey-personalization-gui/Manifest]:
```
!!! Fetched file: yubikey-personalization-gui-3.1.25.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
```

Comment (jok-ts) [reply sys-auth/yubikey-personalization-gui/Manifest]: Dammit, the file linked in the [yubikey site](https://www.yubico.com/support/knowledge-base/categories/articles/yubikey-personalization-tools/) isn't the same they have in github. 
I'll check the differences before pushing a change.

Comment (jok-ts) [reply sys-auth/yubikey-personalization-gui/Manifest]: The tarball I had included a ChangeLog, the github tarball doesn't. No other changes.

**Comment by a17r** [comment]
Source dir is not fixed yet.

**Comment by jok-ts** [comment]
This is so not my night, sorry for the churn.
S="${WORKDIR}/${PN}-${P}" makes no sense but that's what they seem to be using for at least the latest couple of versions, so let's go with that.

**Comment by a17r** [comment]
Thanks! I only added one missing dev-qt dependency, merged. :)

**Comment by jok-ts** [comment]
Thanks, @a17r ! 
I have to say github and gentoo are a match made in heaven when it comes to scratching an itch.

**Pull request closed without merging**